src/core/core/logging.cpp

```diff
@@ -39,13 +39,29 @@ DSN_DEFINE_string("core",
 
 DSN_DEFINE_bool("core", logging_flush_on_exit, true, "flush log when exit system");
 
+namespace dsn {
+std::function<std::string()> log_prefixed_message_func = []() {
+    int tid = dsn::utils::get_current_tid();
+    char res[25];
+    sprintf(res, "unknown.io-thrd.%05d: ", tid);
+    return std::string(res);
```

Comment:
Could you use a static thread_local variable for this prefix? It's not neede to create the string every time.